homeassistant/components/shelly/sensor.py

```diff
@@ -53,6 +54,18 @@
         value=lambda value: round(value, 1),
         device_class=sensor.DEVICE_CLASS_POWER,
     ),
+    ("emeter", "voltage"): BlockAttributeDescription(
+        name="Voltage",
+        unit=VOLT,
+        value=lambda value: round(value, 1),
+        device_class=sensor.DEVICE_CLASS_VOLTAGE,
+    ),
+    ("emeter", "powerFactor"): BlockAttributeDescription(
```

Comment:
What is this value?

Reply:
https://en.wikipedia.org/wiki/Power_factor

Reply:
As powerfactor has a range 0-1, can we show it as a float like it is shown in the app ?

Reply:
Good point, I will change this tommorow.

Reply:
However, the default power factor unit in HA is percent https://github.com/home-assistant/core/blob/dev/homeassistant/components/sensor/__init__.py#L48

Reply:
I would go with HA standard then; in the end there is not that difference between the two representations.